allensdk/brain_observatory/behavior/data_objects/trials/trials.py

```diff
@@ -0,0 +1,192 @@
+import warnings
```

Comment:
I just did a double-take at the existence of `trial.py` and `trials.py`. Can we rename this module (and its flagship class) something like `TrialSet` or `TrialList`?

Reply:
Alternatively, can rename `Trial` `SingleTrial`, if it is important that `Trials` remain `Trials`?

Reply:
I went with renaming `Trials` -> `TrialTable`